### PR TITLE
set up tests

### PR DESCRIPTION
**Pull request by JessLayton**

### CODE REVIEW

File: backend/test/ddpoTest.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        } else {
+          res.should.have.status(200);
+          res.body.data.should.be.a('array');
+          res.body.data.should.not.have.a.lengthOf(0);
+          done();
+        }
+      });
+  });
+ 
+});
```
Comment (lukecottenham) [inline backend/test/ddpoTest.js]: newline needed here pls

File: backend/test/hooks.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require('dotenv').config();
```
Comment (lukecottenham) [inline backend/test/hooks.js:1]: Is this used?

**Review by lukecottenham — CHANGES_REQUESTED** [review]

File: backend/test/hooks.js (hunk 2 above)
Comment (JessLayton) [reply backend/test/hooks.js:1]: Think its needed - I get a Mongoose error: _The uri parameter to openUri() must be a string, got "undefined"_ if I remove it

Comment (lukecottenham) [reply backend/test/hooks.js:1]: Fair enough, maybe look into why it's doing that separately then

**Review by lukecottenham — APPROVED** [review]

**Pull request merged**